### DISCUSSION TITLE
Can results be posted/discussed as the YouTube videos are still being released?

### DISCUSSION THREAD

**Discussion by mattheww95** [PlummersSoftwareLLC/Primes#263] [Q&A] [answered]
I ran the code in the repository, and I was left quite surprised by the results. I would really like to know more about why a certain language did so well and discuss the outcomes here. 

But I am aware Dave is still releasing his video series on the results, and I would not want to unintentionally spoil anything. So I was wondering can I post the results of the different prime sieves in the discussion area of this GitHub?

**Comment by rbergen** [comment, marked as answer]
I'm comfortable with saying that Dave will support whatever leads to a better understanding of the languages in this project. I think that open discussions about results and how they came to be will definitely contribute to that.
What is important to remember is that not every machine will yield the same (order of) results, every time. On the one hand, that means that results posted here do not interfere with Dave's video series. On the other, it means those same results don't invalidate his final ranking either, if it differs.

**Reply by mattheww95** [reply]
Thank you for the clarification!

**Comment by Google987** [comment]
https://youtu.be/KLm6Y_tmQuE

made a small video about this repo. Hope that's fine.
It's in Hindi btw.

Thanks to everyone who's contributed to this repo. It's an awesome project.
